app/pages/lab/subject-set.cjsx

```diff
@@ -13,9 +13,9 @@ isAdmin = require '../../lib/is-admin'
 
 NOOP = Function.prototype
 
-VALID_SUBJECT_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg']
+VALID_SUBJECT_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg', '.mp3']
 INVALID_FILENAME_CHARS = ['/', '\\', ':', ',']
-MAX_FILE_SIZE = 600 * 1024
+MAX_FILE_SIZE = 4000 * 1024
```

Comment:
Do we really want to bump the file size this much? 4MB seems too big for downloading with preload to auto, especially on mobile devices.